client/index.html

```diff
@@ -79,805 +79,10 @@ <h1 class="title">The Lounge is loading…</h1>
-						</div>
-
-						<div class="help-item">
-							<div class="subject">
-								<code>/quit [message]</code>
-							</div>
-							<div class="description">
-								<p>
-									Disconnect from the current network with an optional message.
-								</p>
-							</div>
-						</div>
-
-						<div class="help-item">
-							<div class="subject">
-								<code>/raw message</code>
-							</div>
-							<div class="description">
-								<p>Send a raw message to the current IRC network.</p>
-								<p>Aliases: <code>/quote</code>, <code>/send</code></p>
-							</div>
-						</div>
-
-						<div class="help-item">
-							<div class="subject">
-								<code>/slap nick</code>
-							</div>
-							<div class="description">
-								<p>Slap someone in the current channel with a trout!</p>
-							</div>
-						</div>
-
-						<div class="help-item">
-							<div class="subject">
-								<code>/topic newtopic</code>
-							</div>
-							<div class="description">
-								<p>Set the topic in the current channel.</p>
-							</div>
-						</div>
-
-						<div class="help-item">
-							<div class="subject">
-								<code>/unban nick</code>
-							</div>
-							<div class="description">
-								<p>Unban (<code>-b</code>) a user from the current channel.
-									This can be a nickname or a hostmask.</p>
-							</div>
-						</div>
-
-						<div class="help-item">
-							<div class="subject">
-								<code>/voice nick [...nick]</code>
-							</div>
-							<div class="description">
-								<p>
-									Give voice (<code>+v</code>) to one or several users in the
-									current channel.
-								</p>
-							</div>
-						</div>
-
-						<div class="help-item">
-							<div class="subject">
-								<code>/whois nick</code>
-							</div>
-							<div class="description">
-								<p>
-									Retrieve information about the given user on the current
-									network.
-								</p>
-							</div>
-						</div>
-
-						<h2>About The Lounge</h2>
-
-						<p class="about">
-							{{#if gitCommit}}
-								The Lounge is running from source
-								(<a href="https://github.com/thelounge/lounge/tree/{{ gitCommit }}" target="_blank" rel="noopener"><code>{{ gitCommit }}</code></a>).<br>
-							{{else}}
-								The Lounge is in version <strong>{{version}}</strong>
-								(<a href="https://github.com/thelounge/lounge/releases/tag/v{{ version }}" target="_blank" rel="noopener">See release notes</a>).<br>
-							{{/if}}
-
-							<a href="https://thelounge.github.io/" target="_blank" rel="noopener">Website</a><br>
-							<a href="https://thelounge.github.io/docs/" target="_blank" rel="noopener">Documentation</a><br>
-							<a href="https://github.com/thelounge/lounge/issues/new" target="_blank" rel="noopener">Report a bug</a>
-						</p>
-					</div>
-				</div>
+				<div id="sign-in" class="window"></div>
+				<div id="connect" class="window"></div>
+				<div id="settings" class="window" data-type="settings"></div>
```

Comment:
I don't think `data-type="settings"` is being used anymore. Not sure if you want to kill it in this PR or later, but something to keep in mind.